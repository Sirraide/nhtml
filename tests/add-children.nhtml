#foo {
    div
}

eval {
    const foo = $('#foo').children
    foo.push(element('div'))
}