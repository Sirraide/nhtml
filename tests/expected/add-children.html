<div id='foo'>
    <div></div>
    <div></div>
</div>
